jobs/mysql_restore_job.go

```diff
@@ -386,5 +401,9 @@ func restoreBackup(ctx context.Context, backupDirectory, mySQLDirectory string)
 		return errors.WithStack(err)
 	}
 
+	if err := os.Chmod(mySQLDirectory, mysqlDirPermissions); err != nil {
+		return errors.Wrap(err, "failed to change permissions for MySQL base directory")
+	}
```

Comment:
I guess it should be recursive as well. Same as we do few lines above with chown, no?

Reply:
To allow all users to connect to the DB, we only need to change the permissions of the parent directory. All permissions inside the main directory are controlled by xtrabackup. So I don't think we need to change it recursively as it works without any problems.

Reply:
I think it's good to add comment into the code why we set these permissions

Reply:
Done.